gradle/scripts/lib/java.gradle

```diff
@@ -108,6 +108,8 @@ configure(projectsWithFlags('java')) {
                 dependsOn: [tasks.checkstyleMain, tasks.checkstyleTest])
 
         test {
+            useJUnitPlatform()
```

Comment:
I think this has to be done in `/build.gradle` for all Java projects instead of here, because `gradle/scripts` is shared with other projects.